docs/pages/joy.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
```

Comment:
This file is for demonstration of using the design system created from `createDesignSystem` api, will be removed before merge